setup/gcp/install_setup.py

```diff
@@ -54,9 +58,19 @@ def run():
                               help='Notification recipient email')
     email_params.add_argument('--gsuite-superadmin-email',
                               help='G Suite super admin email')
-
     args = vars(parser.parse_args())
-    forseti_setup = gcloud_env.ForsetiGcpSetup(**args)
+
+    if not args.get('type'):
+        # If the user didn't specify a type, install both server and client
+        ForsetiServerInstaller(**args).run_setup()
+        ForsetiClientInstaller(**args).run_setup()
+        return
+
+    if args.get('type') == 'server':
+        forseti_setup = ForsetiServerInstaller(**args)
+    else:
```

Comment:
Should the final ```else``` here be a check that a type is actually specified?  i.e. should this ```else``` be to abort the install if type is not specified?